app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -126,14 +128,21 @@ def inspect_work
       presenter
     end
 
-    #    attr_writer :actor
+    def manifest
+      headers['Access-Control-Allow-Origin'] = '*'
+      render json: manifest_builder.to_h
```

Comment:
Is it more appropriate/idiomatic to use `#as_json` instead of `#to_h`.

Reply:
@jeremyf I believe we need the `#to_h` here because of: https://github.com/samvera-labs/iiif_manifest/blob/35f2f17d80034aab13431b7f6350e612fb1db020/lib/iiif_manifest/manifest_builder.rb#L31-L33

See also this difference:

```ruby
# 2.4.1 :040 > ApplicationController.render json: ::IIIFManifest::ManifestFactory.new(Thing.new('foobar')).as_json
"{\"work\":{\"id\":\"foobar\"},\"builders\":{\"factories\":[{},{\"hash_args\":{\"canvas_builder_factory\":{\"composite_builder\":{},\"canvas_builder_factory\":{\"hash_args\":{\"iiif_canvas_factory\":{},\"image_builder\":{\"hash_args\":{\"iiif_annotation_factory\":{},\"resource_builder_factory\":{\"hash_args\":{\"iiif_resource_factory\":{},\"image_service_builder_factory\":{\"hash_args\":{\"iiif_service_factory\":{}},\"factory\":{}}},\"factory\":{}}},\"factory\":{}}},\"factory\":{}}},\"sequence_factory\":{}},\"factory\":{}},{\"hash_args\":{\"canvas_builder_factory\":{\"hash_args\":{\"iiif_canvas_factory\":{},\"image_builder\":{\"hash_args\":{\"iiif_annotation_factory\":{},\"resource_builder_factory\":{\"hash_args\":{\"iiif_resource_factory\":{},\"image_service_builder_factory\":{\"hash_args\":{\"iiif_service_factory\":{}},\"factory\":{}}},\"factory\":{}}},\"factory\":{}}},\"factory\":{}},\"iiif_range_factory\":{}},\"factory\":{}}],\"composite_builder\":{}},\"top_record_factory\":{}}" 

# 2.4.1 :041 > ApplicationController.render json: ::IIIFManifest::ManifestFactory.new(Thing.new('foobar')).to_h
"{\"@context\":\"http://iiif.io/api/presentation/2/context.json\",\"@id\":\"http://example.org/foobar/manifest\",\"@type\":\"sc:Manifest\",\"label\":\"#\\u003cThing:0x00000004513b60\\u003e\",\"description\":\"foo\"}" 
```

Reply:
Thanks for the follow-up. That closes my concern.